types/weather.go

```diff
@@ -54,6 +54,7 @@ type Sys struct {
 // WeatherResponse is the struct that encodes the API data from our weather API
 type WeatherResponse struct {
 	BaseMessage
```

Comment:
Why is `WeatherResponse` embedding `BaseMessage`? None of the other API response types do

Reply:
This was something we were doing a while ago, seems like its still here. I don't think we need it anymore